modules/market/src/main/java/com/opengamma/strata/market/observable/Quote.java

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.observable;
+
+import org.joda.beans.ImmutableBean;
+import org.joda.beans.gen.BeanDefinition;
+import org.joda.beans.gen.PropertyDefinition;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import org.joda.beans.Bean;
+import org.joda.beans.BeanBuilder;
+import org.joda.beans.JodaBeanUtils;
+import org.joda.beans.MetaBean;
+import org.joda.beans.MetaProperty;
+import org.joda.beans.impl.direct.DirectMetaBean;
+import org.joda.beans.impl.direct.DirectMetaProperty;
+import org.joda.beans.impl.direct.DirectMetaPropertyMap;
+import org.joda.beans.impl.direct.DirectPrivateBeanBuilder;
+
+import com.opengamma.strata.basics.StandardId;
+
+/**
+ * A single value for a given security, represented by a quote, such as 'EUR/GBP 1W FX Opt ATM Straddle' - 10.0.
+ * <p>
+ *
+ * This represents a single numeric value of a given security, such as the mid point between the buy/sell.
+ * <p>
+ * See {@link QuoteId} fot the representation that does not contain a value.
```

Comment:
fot -> for, although I'm not sure this line is adding anything, so best to remove